app/services/mptr/school_eligibility.rb

```diff
@@ -0,0 +1,78 @@
+module Mptr
```

Comment:
As discussed, we're going to name this module `MathsAndPhysics`. MPTR as an acronym isn't part of anyone's vocabulary and we can't think of a worthwhile reason for shortening it beyond making it consistent with the `TSLR` module (which is part of our vocabulary) and we're valuing clarity and ease of understanding over consistency and brevity.